### DISCUSSION TITLE
Nonsense Code Quality

### DISCUSSION THREAD

**Discussion by doydevlabs** [community/community#38644] [Copilot Conversations]
Hello,
What a magical and an amazing product indeed! 💯 🥇 

However, lately I've started to noticed the AI started to generated garbage and endless loops, any suggestions on how to improve this?

![Screenshot 2022-11-09 at 9 01 39 AM](https://user-images.githubusercontent.com/114758361/200752344-c5f0ad1c-8313-4808-871b-24180b6dfdd8.jpg)

**Comment by ossanna16** [comment]
Hi there @doydevlabs and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment]
@doydevlabs Hello, can you provide the following detail:

1. What version of the copilot extension are you using?
2. What steps are you performing that cause endless loops to be generated?
